core/src/main/java/edu/wpi/grip/core/operations/composite/PublishVideoOperation.java

```diff
@@ -44,14 +45,14 @@
           .build();
   private static final int PORT = 1180;
   private static final byte[] MAGIC_NUMBER = {0x01, 0x00, 0x00, 0x00};
-  private static int numSteps = 0;
-  private final Logger logger = Logger.getLogger(PublishVideoOperation.class.getName());
+
+  private static int numSteps;
   private final Object imageLock = new Object();
   private final BytePointer imagePointer = new BytePointer();
   private final Thread serverThread;
   private final InputSocket<Mat> inputSocket;
   private final InputSocket<Number> qualitySocket;
-  private volatile boolean connected = false;
+  private volatile boolean connected;
```

Comment:
TBH this should probably be refactored into a singleton with a static `getInstance()` method instead of counting created instances...